src/components/molecules/TCNavbar/TCNavbar.tsx

```diff
@@ -0,0 +1,29 @@
+import { FunctionComponent, useState } from 'react'
+import { NavbarItemModel } from '../../../model/NavbarItemModel'
+import { TCNavbarItem } from '../../atoms/TCNavbarItem/TCNavbarItem'
+
+interface TCNavbarProps {
+  items: NavbarItemModel[]
+}
+
+export const TCNavbar: FunctionComponent<TCNavbarProps> = ({ items }) => {
+  const [selectedItemIndex, setSelectedItemIndex] = useState(0)
+
+  const handleOnClick = (itemIndex: number) => {
+    setSelectedItemIndex(itemIndex)
+  }
+
+  return (
+    <div>
+      {items.map((item, index) => (
+        <TCNavbarItem
```

Comment:
If you wrap this component in <li tag you can add key attribute to it to help react track item rendering:
https://reactjs.org/docs/lists-and-keys.html

Then div above should be changed to <ul.